src/MemberService/Auth/Email/SlowEmailSender.cs

```diff
@@ -0,0 +1,31 @@
+using MemberService.Configs;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+using System.Threading.Tasks;
+
+namespace MemberService.Auth.Email
+{
+    public class SlowEmailSender : ISlowEmailSender
+    {
+        private readonly EmailConfig _config;
+        private readonly SendGridClient _client;
+
+        public SlowEmailSender(EmailConfig config)
+        {
+            _config = config;
+            _client = new SendGridClient(_config.SendGridApiKey);
```

Comment:
Maybe inject this object, and make it a singleton in the ConfigureServices?

Reply:
Fixed. Changed to Scoped, as the docs don't make any recommendations and singletons are scary